### PR TITLE
[UI] - Fixed YAML text overlapping

### LINKED ISSUE #5811: [UI] Design Configurator: Overlapping text in code view
#### Current Behavior
Yaml/text overlapping in the code viewer of the Design Configurator - http://localhost:9081/configuration/patterns

#### Screenshots/Logs
Screen Shot 2022-07-13 at 6 49 53 AM

#### Environment

- Browser: Safari 

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by subhradwip01**
**Description**

This PR fixes #5811

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by subhradwip01** [comment]
Output:

File: ui/components/FlipCard.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ function GetChild(children, key) {
 }
 
 function FlipCard({
-  classes, duration = 500, onClick, children
+  classes, duration = 500, onClick,onShow, children
```
Comment (Chadha93) [inline ui/components/FlipCard.js:29]:
```suggestion
  classes, duration = 500, onClick, onShow, children
```

File: ui/components/MesheryApplications/ApplicationsCard.js [hunk 2]
```diff
@@ -37,6 +37,7 @@ function MesheryApplicationCard({
   }
   const [gridProps, setGridProps] = useState(INITIAL_GRID_SIZE);
   const [fullScreen, setFullScreen] = useState(false);
+  const [showCode,setShowCode]=useState(false);
```
Comment (Chadha93) [inline ui/components/MesheryApplications/ApplicationsCard.js:40]:
```suggestion
  const [showCode, setShowCode]=useState(false);
```

File: ui/components/MesheryPatterns/MesheryPatternCard.js [hunk 3]
```diff
@@ -38,11 +38,14 @@ function MesheryPatternCard({
   }
   const [gridProps, setGridProps] = useState(INITIAL_GRID_SIZE);
   const [fullScreen, setFullScreen] = useState(false);
+  const [showCode,setShowCode]=useState(false);
```
Comment (Chadha93) [inline ui/components/MesheryPatterns/MesheryPatternCard.js:41]: here also space after the comma, @subhradwip01 did you used no-verify the linter should format it automatically?

File: ui/components/MesheryFilters/FiltersCard.js [hunk 4]
```diff
@@ -60,6 +61,7 @@ function FiltersCard({
           setGridProps(INITIAL_GRID_SIZE)
         }}
         duration={600}
+        onShow={() => setTimeout(() => setShowCode(p => !p),500)}
```
Comment (Chadha93) [inline ui/components/MesheryFilters/FiltersCard.js:64]: Could we provide anything other than p? More readable/understandable by others.

**Review by Chadha93 — CHANGES_REQUESTED** [review]
@subhradwip01 good work, some minors to look at.

**Comment by Chadha93** [comment]
Restarting the failed workflows.

**Comment by leecalcote** [comment]
All greens ✅

File: ui/components/MesheryPatterns/MesheryPatternCard.js (hunk 3 above)
Comment (subhradwip01) [reply ui/components/MesheryPatterns/MesheryPatternCard.js:41]: i have not used no-verify

**Comment by subhradwip01** [comment]
@Chadha93 any command for this code formatting or I need to do it manually?

**Comment by Chadha93** [comment]
> @Chadha93 any command for this code formatting or I need to do it manually?

Ok, try manually once.

**Comment by subhradwip01** [comment]
> > @Chadha93 any command for this code formatting or I need to do it manually?
> 
> Ok, try manually once.

ok

File: ui/components/MesheryApplications/ApplicationsCard.js [hunk 5]
```diff
@@ -37,6 +37,7 @@ function MesheryApplicationCard({
   }
   const [gridProps, setGridProps] = useState(INITIAL_GRID_SIZE);
   const [fullScreen, setFullScreen] = useState(false);
+  const [showCode, setShowCode]=useState(false);
```
Comment (adithyaakrishna) [inline ui/components/MesheryApplications/ApplicationsCard.js:40]:
```suggestion
  const [showCode, setShowCode] = useState(false);
```

File: ui/components/MesheryFilters/FiltersCard.js [hunk 6]
```diff
@@ -34,6 +34,7 @@ function FiltersCard({
   }
   const [gridProps, setGridProps] = useState(INITIAL_GRID_SIZE);
   const [fullScreen, setFullScreen] = useState(false);
+  const [showCode, setShowCode]=useState(false);
```
Comment (adithyaakrishna) [inline ui/components/MesheryFilters/FiltersCard.js:37]:
```suggestion
  const [showCode, setShowCode] = useState(false);
```

File: ui/components/MesheryPatterns/MesheryPatternCard.js [hunk 7]
```diff
@@ -38,11 +38,14 @@ function MesheryPatternCard({
   }
   const [gridProps, setGridProps] = useState(INITIAL_GRID_SIZE);
   const [fullScreen, setFullScreen] = useState(false);
+  const [showCode, setShowCode]=useState(false);
```
Comment (adithyaakrishna) [inline ui/components/MesheryPatterns/MesheryPatternCard.js:41]:
```suggestion
  const [showCode, setShowCode] = useState(false);
```

File: ui/components/MesheryPatterns/MesheryPatternCard.js [hunk 8]
```diff
@@ -38,11 +38,14 @@ function MesheryPatternCard({
...
+
 
   const toggleFullScreen = () => {
     setFullScreen(!fullScreen);
   };
 
+
```
Comment (adithyaakrishna) [inline ui/components/MesheryPatterns/MesheryPatternCard.js:48]:
```suggestion
```

**Review by adithyaakrishna — CHANGES_REQUESTED** [review]
@subhradwip01 Could you please address the above changes?

**Comment by subhradwip01** [comment]
> @subhradwip01 Could you please address the above changes?

Ya.. I will

**Review by Chadha93 — APPROVED** [review]
Good Work @subhradwip01

**Comment by Chadha93** [comment]
Tested OK, merging this one, please do have a look at this issue https://github.com/meshery/meshery/issues/5862

**Comment by adithyaakrishna** [comment]
Yayyy @subhradwip01  🎉

**Pull request merged**